### ISSUE TITLE
Uncaught TypeError: _this.notificationEl.querySelector is not a function

### ISSUE THREAD

**Issue by faliagas** [AtomLinter/linter-gcc#209]
Steps to reproduce:

1. Open atom from command line: atom myfile.cpp

**Atom**: 1.40.1 x64
**Electron**: 3.1.10
**OS**: Linux Mint
**Thrown From**: [linter-gcc](https://github.com/hebaishi/linter-gcc) package 0.7.1

### Stack Trace

Uncaught TypeError: _this.notificationEl.querySelector is not a function

```
At /home/apostol/.atom/packages/linter-gcc/node_modules/atom-package-deps/lib/view.js:22

TypeError: _this.notificationEl.querySelector is not a function
    at /packages/linter-gcc/node_modules/atom-package-deps/lib/view.js:22:58
```

### Commands

```
```

### Non-Core Packages

```
atom-overtype-mode 0.4.0 
gruvbox-plus-syntax 1.3.1 
language-autotools 2.0.1 
language-meson 0.2.1 
language-vala 0.2.0 
linter-gcc 0.7.1 
monokai 0.24.0 
monokai-shade 0.4.0 
project-manager 3.3.8 
```

**Comment by Arcanemagus** [comment]
Using this issue to gather the related ones together. Some suggested troubleshooting steps:
* Make sure you have the `linter` and `linter-ui-default` packages installed
* Make sure you have `gcc` installed and available on your `PATH`

Tagging this as a bug since it should be handled better in any case.

**Comment by Arcanemagus** [comment]
Looks like this was probably fixed in #137.
